docs/nx/azure-react/generator-app-insights-web.md

```diff
@@ -0,0 +1,102 @@
+### @ensono-stacks/azure-react:app-insights-web
+
+<details>
+<summary>Installs and configures a ReactJS app with App Insights.</summary>
+
+This generator will add and configure [applicationinsights reactjs](https://www.npmjs.com/package/@microsoft/applicationinsights-react-js) and [applicationinsights web](https://www.npmjs.com/package/@microsoft/applicationinsights-web) npm packages for you.
+
+This enables the following:
+
+- Tracking of router changes
+- React components usage statistics
+
+It requires the connection string to be set in an env variable, from which it will be read.
```

Comment:
Could we move this into the usage sections and then have an admonition stating exactly what env variable needs to be set?
E,g

:::caution

App insights requires the `X` environment variable to be set to......x?

:::

Reply:
Added the admonition to explain how to set the env variable and a link to the MS docs regarding connection strings